_data/retail.yml

```
@@ -279,13 +326,20 @@ websites:
     twitter: mediamarkt_de
     facebook: mediamarkt
     lang: de
+    regions:
+      - de
 
   - name: Mercado Libre
     url: https://www.mercadolibre.com
     img: mercado-libre.png
     tfa:
       - sms
       - proprietary
+    regions:
+      - ar
+      - mx
+      - co
+      - br
```

```suggestion
      - proprietary
```
Mercado Libre [serves 18 countries](http://mercadolibre.com/). I think we should remove the region tag as this is a large number of regions to tag.